docker-compose.yml

```diff
@@ -1,12 +1,23 @@
 version: "3.9"
 services:
   daemons:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile_Daemons 
     stdin_open: true
-    network_mode: host
     env_file:
-      - .env
+      - .env.example
```

Comment:
We should keep `.env`, since we may want to change the variables for local development. And let the `.env.example` only as an example.
Maybe add "run `cp .env.example .env` before running locally" on `README.md`